apps/sensor-default/src/main.c

```diff
@@ -435,6 +452,9 @@ void request_on_data_received(struct blecon_request_receive_data_op_t* receive_d
 }
 
 void on_connection(struct blecon_t* blecon) {
+#ifdef CONFIG_BLECON_LIB_LED
+    blecon_led_set_connection_state(blecon_led_connection_state_connected);
```

Comment:
So the worry I have is that with all these flashy LED indicators will consume power when the devices are in storage.  I think this problematic for factory-fresh devices as well as demo devices in storage. We can always config BLECON_LIB_LED=f but that would also mean announce LED isn't available.

Reply:
Makes sense! I made some changes to only enable LEDs for a while (defaults to 12 hours) upon pressing the button.